modules/tflite-micro/CMakeLists.txt

```
@@ -14,6 +15,14 @@ if(CONFIG_TENSORFLOW_LITE_MICRO)
     ${TENSORFLOW_LITE_MICRO_DIR}/third_party_static/ruy
   )
 
+  if (CONFIG_TENSORFLOW_LITE_MICRO_CMSIS_NN_KERNELS)
+    set(CMSIS_NN_OPTIMIZED_KERNEL_DIR cmsis_nn)
+    set(tflm_cmsis_nn_glue_path ${ZEPHYR_CMSIS_MODULE_DIR})
+
+    zephyr_include_directories(${tflm_cmsis_nn_glue_path})
+    add_definitions(-DCMSIS_NN)
```

also assuming `-DCMSIS_NN` is only necessary for building the "tflite-micro library," this should use `zephyr_library_compile_definitions`.

```suggestion
    zephyr_library_compile_definitions(CMSIS_NN)
```